templates/user/AccountSettingsCompany.html

```diff
@@ -0,0 +1,14 @@
+{% extends "user/AccountSettings.html" %}
+
+{% block settings_content %}
+    <div id="company_settings_content">
+        <div class="card">
+            <div class="card-content">
+                <p class="flow-text center-align services-title">
+                    You haven't added company in which you are working.
+                    You can easily do it. (we are working on this)
```

Comment:
From my point of view such descriptions help people understand what to. Nicely done!